mushr_control/XYHVPath path
---
bool success
